src/txn/txn_rollback_to_stable.c

```diff
@@ -8,6 +8,7 @@
 
 #include "wt_internal.h"
 
+static uint64_t verb_flags;
```

Comment:
This approach relies on a file scoped global being set by __wt_rollback_to_stable() and other functions need to have been called after the variable has been set through __wt_rollback_to_stable(). This could lead to unexpected bugs in the future if the calling order breaks. Also, I'm not too fond of global variables.

As an alternative I have the following suggestion:
We can replace
`__wt_verbose(session, WT_VERB_RTS,`
with
`__wt_verbose(session,
  F_ISSET(S2C(session), WT_CONN_RECOVERING) ? WT_VERB_RECOVERY | WT_VERB_RTS : WT_VERB_RTS,`

We can put that whole ternary-operator-if-condition in a macro if it helps, but it will be okay if we don't.

Reply:
Yes Sulabh, that makes sense! Thank you.